mne_bids/io.py

```diff
@@ -83,3 +92,72 @@ def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
         raise ValueError('Raw file name extension must be one of {}\n'
                          'Got {}'.format(ALLOWED_EXTENSIONS, ext))
     return raw
+
+
+def read_raw_bids(bids_fname, output_path, return_events=True,
+                  verbose=True):
+    """Read BIDS compatible data.
+
+    Parameters
+    ----------
+    bids_fname : str
+        Full name of the data file
+    output_path : str
+        Root of the BIDS folder
+    return_events: bool
+        Weather to return events or not. Default is True.
+    verbose : bool
+        The verbosity level
+    """
+    from .utils import _parse_bids_filename
+
+    # Full path to data file is needed so that mne-bids knows
+    # what is the modality -- meg, eeg, ieeg to read
+    bids_basename = '_'.join(bids_fname.split('_')[:-1])
+    kind = bids_fname.split('_')[-1].split('.')[0]
+    _, ext = _parse_ext(bids_fname)
+
+    params = _parse_bids_filename(bids_basename, verbose)
+    meg_dir = op.join(output_path, 'sub-%s' % params['sub'],
```

Comment:
why call the variable `meg_dir`? This could be EEG or iEEG data as well, right? I suggest `kind_dir`